apps/challenges/urls.py

```diff
@@ -25,4 +25,7 @@
         name='get_challenges_based_on_teams'),
     url(r'(?P<challenge_pk>[0-9]+)/challenge_phase_split$', views.challenge_phase_split_list,
         name='challenge_phase_split_list'),
+    url(r'challenge/challenge_host_team/(?P<challenge_host_team_pk>[0-9]+)/zip/$',
```

Comment:
I am actually not clear of whether we should take challenge host team as a paramter in the api or not. Writing it down here, so that I dont forget. Will update you over this.